governance/votes/December 2024 out-of-schedule spell.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Template - [Executive Vote] Out-of-schedule Executive Vote - Stability Fee Changes, Savings Rate Changes - December $day, 2024
+summary: Increase Stability Fees for multiple vault types, increase Savings Rates.
+date: 2024-12-DDT00:00:00.000Z
```

Comment:
![image](https://github.com/user-attachments/assets/941f13ee-d4b4-4f81-ba2f-b379fc1a4805)

This returns an Invalid Date when put through https://vote.makerdao.com/executive/create

We suggest using a placeholder date and making clear in the text that this is the case.